### PR TITLE
app doc update for 7 v2 apps - release 3

### PR DESCRIPTION
**Pull request by shivani-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/saas-cloud/proofpoint-tap.md [hunk 1]
```diff
@@ -161,9 +161,7 @@ To set up [Cloud to Cloud Integration Proofpoint TAP Source](/docs/send-data/hos
 
 ## Installing the Proofpoint TAP app
 
-This section provides instructions on how to install the Proofpoint TAP app, as well as examples of each of the dashboards. The app pre-configured searches and dashboards provide easy-to-access visual insights into your data.
-
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Proofpoint TAP dashboards
```
Comment (JV0812) [inline docs/integrations/saas-cloud/proofpoint-tap.md:167]: Can you delete this content and just add the import component?

Comment (kimsauce) [reply docs/integrations/saas-cloud/proofpoint-tap.md:167]: @shivani-sumo

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by JV0812** [comment]
The intent of this PR is covered in the PR https://github.com/SumoLogic/sumologic-documentation/pull/3434.

**Comment by JV0812** [comment]
The intent of this PR is covered in the PR https://github.com/SumoLogic/sumologic-documentation/pull/3434.

**Pull request closed without merging**